.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "es2015",
+    "stage-2"
```

Comment:
I don't see anything used that would require any of the `stage-` presets. Just es2015 should be fine. Did I miss something?